advanced/creating-a-skin.md

```diff
@@ -4,48 +4,161 @@ title: Create a skin for TinyMCE
 
-## Modifying the icons
+### Creating a skin
+
+1. In the `/oxide` directory, begin by duplicating the `default` folder located in `src/less/skins/ui/` and rename it to the name of your skin.
+1. Open the file `src/less/theme/globals/global-variables.less` and **copy** the variables to change into the `skin.less` file of the duplicated folder in the previous step.
+1. Change the values. The variables in the `skin.less` will override the default values.
+
+For more detailed customizations, review the variables in each component, such as `src/less/theme/components/toolbar-button.less` and copy the desired ones to the `skin.less`.
+
+To style interface elements in the content, such as selection color, drag handles, table of contents, bookmarks etc, copy variables to `src/less/skins/ui/<skin-name>/content.less` or `content.inline.less` (if inline mode is being used).
+
+Following is an example of a sample `skin.less` file:
+
+```js
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+@import 'src/less/theme/theme';
+
+//
+
+// Root variables
+// Begin customization by changing these variables as most other variables are derivatives of these.
+@background-color: #1976D2;
+@base-value: 24px;
+@color-black: #222f3e;
+@color-tint: #3498db;
+@color-white: #fff;
+@color-error: #c00;
+@font-stack: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
 
-The icons shipped with the default skin are created by the [IcoMoon](http://icomoon.io/) project and are embedded as web fonts. This makes them *retina ready* and easy to skin.
+// Colors
+@border-color: darken(@background-color, 20);
+@text-color: contrast(@background-color, @color-black, @color-white);
+@text-color-muted: contrast(@background-color, fade(@color-black, 60%), fade(@color-white, 50%));
+```
 
-Import the existing TinyMCE icons by uploading the "icomoon.dev.svg" file and then adding and/or changing the icons that are provided. Alter the `icons.less` file to add more icons.
+### Preview the skin
+
+1. To preview the new skin, run the following command from a terminal:
+```js
+npm run start
+```
+2. Navigate to the address shown in the terminal, usually `localhost:3000`. The TinyMCE skin SDK for Silver theme development environment page will appear in the browser.
+3. In the **Editor examples** section, select the new skin from the **UI skin** drop-down menu.
+![**UI skin**]({{site.baseurl}}/images/default-copy.png)
+
+#### Result
+
+TinyMCE editor with all the configured skin changes will appear in the browser screen.
```

Comment:
I just realised - did we ever say how to actually use the skin in the editor? We should link to skin and skin_url - https://www.tiny.cloud/docs/configure/editor-appearance/#skin_url

Reply:
Added the url reference into the moving skin into TinyMCE section.